Services/Common/OJS.Services.Common/IContestsActivityService.cs

```diff
@@ -8,9 +9,13 @@ public interface IContestsActivityService : IService
 {
     Task<IContestActivityServiceModel> GetContestActivity(int id);
```

Comment:
Why are you using an interface here, instead of the class ContestActivityServiceModel ?

Reply:
The contest activity service was implemented this way, so i am just following the implementation of the initial creator. Returning an instance of an implementation achieves abstraction.

Reply:
I find this implementation to be really odd, but maybe that is just me.